x-pack/libbeat/management/tests/mbtest/system/process_integration_test.go

```diff
@@ -0,0 +1,220 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration
+
+package tests
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/elastic/beats/v7/libbeat/common/reload"
+	lbmanagement "github.com/elastic/beats/v7/libbeat/management"
+	"github.com/elastic/beats/v7/x-pack/libbeat/management"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/elastic/elastic-agent-client/v7/pkg/client"
+	"github.com/elastic/elastic-agent-client/v7/pkg/client/mock"
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+
+	"github.com/elastic/beats/v7/x-pack/libbeat/management/tests"
+	"github.com/elastic/beats/v7/x-pack/metricbeat/cmd"
+
+	conf "github.com/elastic/elastic-agent-libs/config"
+)
+
+func TestProcessStatusReporter(t *testing.T) {
+	unitOneID := mock.NewID()
+	unitOutID := mock.NewID()
+	token := mock.NewID()
+
+	tests.InitBeatsForTest(t, cmd.RootCmd)
+
+	filename := fmt.Sprintf("test-%d", time.Now().Unix())
+	outPath := filepath.Join(os.TempDir(), filename)
+	t.Logf("writing output to file %s", outPath)
+	err := os.Mkdir(outPath, 0775)
+	require.NoError(t, err)
+	defer func() {
+		err := os.RemoveAll(outPath)
+		require.NoError(t, err)
+	}()
+
+	/*
+	 * process with pid=-1 doesn't exist. This should degrade the input for a while */
+	inputStreamIncorrectPid := getInputStream(unitOneID, -1, 1)
+	/*
+	 * process with valid pid. This should change state to healthy */
```

Comment:
We don't have the habit of using `/* ...*/` for comments. The usual on the repo is just to use `//`
If you really want to use `/*  */`, please finish it on a new line:
```suggestion
	/*
	 * process with pid=-1 doesn't exist. This should degrade the input for a while 
	 */
	inputStreamIncorrectPid := getInputStream(unitOneID, -1, 1)
	/*
	 * process with valid pid. This should change state to healthy
	 */
```

However I'd really ask to use `//` here. It gets hard to see the code in-between the comments here:
```suggestion
	// process with pid=-1 doesn't exist. This should degrade the input for a while 
	inputStreamIncorrectPid := getInputStream(unitOneID, -1, 1)
	
	// process with valid pid. This should change state to healthy
```